apps/ff-site/package.json

```diff
@@ -8,7 +8,7 @@
     "prebuild": "tsx scripts/convertYamlToJson.ts && node scripts/injectEncryptionAccessKeyInCMS.js",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint --fix",
+    "lint": "eslint src --fix",
```

Comment:
`eslint --fix` was outputting warnings from build files, I tried adding ignore files to `eslint.config.js` but that didn't work, so I changed the script to `eslint src --fix`